src/scripts/loqui/bindings.js

```diff
@@ -124,17 +124,32 @@ $('section#chat article#main div#text').on('keydown', function (e) {
       $('section#chat nav#plus').removeClass('show');
       Messenger.csn('paused');
     }
+    // if user wants to hide keyboard in landscape
+    // user can clear the box and press backspace to hide it
+    if (this.textContent.length === 0 && e.which == 8 && window.matchMedia('(orientation:landscape)').matches) {
+      $('section#chat article#main div#text').blur();
+    }
   } else {
     $('section#chat article#main button#plus').hide();
     $('section#chat article#main button#say').show();
-    $('section#chat nav#plus').addClass('show');
+    // only show plus bar when in portrait mode
+    if (window.matchMedia('(orientation:portrait)').matches) {
+      $('section#chat nav#plus').addClass('show');
+    } else {
+      $('section#chat nav#plus').removeClass('show');
+    }
     var ul = $('section#chat ul#messages');
     ul[0].scrollTop = ul[0].scrollHeight;
     Messenger.csn('composing');
   }
 }).on('tap', function (e) {
   Lungo.Router.article('chat', 'main');
-  $('section#chat nav#plus').addClass('show');
+  // only show plus bar when in portrait mode
+  if (window.matchMedia('(orientation:portrait)').matches) {
+    $('section#chat nav#plus').addClass('show');
+  } else {
+    $('section#chat nav#plus').removeClass('show');
+  }
```

Comment:
As I said, this is bad style because it is not 'just layout'. It's not about changing a color. It disables a function otherwise accessible in a specific case. This is best done in the view or model, because it documents the desired behaviour, leading to maintainable code. In a few months, nobody would remember why the !important is there. There is not another single !important statement in the CSS (for a reason?)  loqui/im schrieb:In src/scripts/loqui/bindings.js:
>      var ul = $('section#chat ul#messages');
>      ul[0].scrollTop = ul[0].scrollHeight;
>      Messenger.csn('composing');
>    }
>  }).on('tap', function (e) {
>    Lungo.Router.article('chat', 'main');
> -  $('section#chat nav#plus').addClass('show');
> +  // only show plus bar when in portrait mode
> +  if (window.matchMedia('(orientation:portrait)').matches) {
> +    $('section#chat nav#plus').addClass('show');
> +  } else {
> +    $('section#chat nav#plus').removeClass('show');
> +  }

why if you do something like: 

@media (max-height: 200px) {
    nav#plus {
       display: none !important;
    }
}

It would be perfectly fine. Yes we need to use important, lungo is really intrusive, we can't change it. 
Your way you need to mess up the application logic with layout behavior. 

—Reply to this email directly or view it on GitHub.